html_css/preprocessors.md

```diff
@@ -17,15 +18,16 @@ Preprocessors (aka precompilers) can make your life much easier by eliminating c
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
-1. Read [Introduction to CSS Preprocessors from Steven Bradley](http://www.vanseodesign.com/css/css-preprocessors/)
```

Comment:
This link works for me. Can you try it again and let me know.

Reply:
Oh now it's fine. At the time the whole website was throwing a request error. They fixed it. Sorry about that